client/src/components/GameArea.js

```diff
@@ -290,7 +305,7 @@ class GameArea extends Component {
             </Container>
           </Col>
           {/* Sidebar for exile,graveyard,hand,library  */}
-          <Col xs="2" className="sidebar-col p-0 m-0">
+          <Col xs="2" className="sidebar-col p-0 m-0" style={displaySidebar}>
```

Comment:
Are you using `displaySidebar` to show/hide the sidebar? If so, you could use a [conditional render](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator) for it.
```
{this.state.isToggleSidebarOn && (
  <Col xs="2" className="sidebar-col p-0 m-0">
    ...
  </Col>
)}
```

Reply:
Nice suggestion, Andrew! Will do!